test/Inflation/runtests.jl

```diff
@@ -0,0 +1,112 @@
+prior_2 = Dict("distribution" => Parameterized(Normal(3.0, 0.5)), "constraint" => no_constraint(), "name" => "uncons_p")
+prior = ParameterDistribution([prior_1, prior_2])
+prior_mean = mean(prior)
+prior_cov = cov(prior)
+
+rng_seed = 42
+rng = Random.MersenneTwister(rng_seed)
+
+initial_ensemble = EKP.construct_initial_ensemble(rng, prior, N_ens)
+
+Δts = [0.5, 0.75, 0.95]
+
+@testset "Inflation" begin
+
+    ekiobj = nothing
+
+    for Δt_i in 1:length(Δts)
+        Δt = Δts[Δt_i]
+
+        # Get inverse problem
+        y_obs, G, Γy, A =
+            linear_inv_problem(ϕ_star, noise_level, n_obs, prior, rng; obs_corrmat = obs_corrmat, return_matrix = true)
+
+        ekiobj = EKP.EnsembleKalmanProcess(
+            initial_ensemble,
+            y_obs,
+            Γy,
+            Inversion();
+            Δt = Δt,
+            rng = rng,
+            failure_handler_method = SampleSuccGauss(),
+        )
+
+        g_ens = G(get_u_final(ekiobj))
+
+        # ensure error is thrown when scaled time step >= 1
+        @test_throws ErrorException EKP.update_ensemble!(ekiobj, g_ens; multiplicative_inflation = true, s = 3.0)
+        @test_throws ErrorException EKP.update_ensemble!(ekiobj, g_ens; additive_inflation = true, s = 3.0)
+
+        # EKI iterations
+        for i in 1:N_iter
+            # Check SampleSuccGauss handler
+            params_i = get_u_final(ekiobj)
+
+            g_ens = G(params_i)
+
+            # standard update
+            EKP.update_ensemble!(ekiobj, g_ens, EKP.get_process(ekiobj))
+            eki_mult_inflation = deepcopy(ekiobj)
+            eki_add_inflation = deepcopy(ekiobj)
+
+            # multiplicative inflation after standard update
+            EKP.multiplicative_inflation!(eki_mult_inflation)
+            # additive inflation after standard update
+            EKP.additive_inflation!(eki_add_inflation)
+
+            # ensure multiplicative inflation preserves ensemble mean
+            @test get_u_mean_final(ekiobj) ≈ get_u_mean_final(eki_mult_inflation) atol = 1e-10
+            # ensure additive inflation approximately preserves ensemble mean
+            @test get_u_mean_final(ekiobj) ≈ get_u_mean_final(eki_add_inflation) rtol = 0.1
+
+            # ensure inflation increases variance of ensemble
+            ϕ_final_standard = get_ϕ_final(prior, ekiobj)
+            ϕ_final_mult_inflation = get_ϕ_final(prior, eki_mult_inflation)
+            ϕ_final_add_inflation = get_ϕ_final(prior, eki_add_inflation)
+            ϕ_mean_final_standard = get_ϕ_mean_final(prior, ekiobj)
+            ϕ_mean_final_mult_inflation = get_ϕ_mean_final(prior, eki_mult_inflation)
+            ϕ_mean_final_add_inflation = get_ϕ_mean_final(prior, eki_add_inflation)
+            standard_update_variance = norm(ϕ_final_standard .- ϕ_mean_final_standard)
+            @test norm(ϕ_final_mult_inflation .- ϕ_mean_final_mult_inflation) > standard_update_variance
```

Comment:
could this be more precise? we have a large ensemble and we know what the variance inflation should be in each case?